### PR TITLE
Jfacevedo endpoints tfhub obj detect

### PR DESCRIPTION
**Pull request by entrpn**

### CODE REVIEW

**Comment by kweinmeister** [comment]
@entrpn I have made comments [here](https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/blob/jfacevedo-endpoints-tfhub-obj-detect/notebooks%2Fcommunity%2Fvertex_endpoints%2Ftf_hub_obj_detection%2Fdeploy_tfhub_object_detection_on_vertex_endpoints.ipynb%2F/discussion), which looks like they aren't getting propagated back to the PR. Can you please take a look when you get a chance? Thanks.

**Comment by kweinmeister** [comment]
Thanks @entrpn!

Re: undeployment, I believe you can actually use this syntax to assign the variable in one step:
`endpoint = !gcloud ai endpoints describe $ENDPOINT_ID --region=$REGION --project=$PROJECT_ID | grep "id:"`

Then, the user won't have to manually edit it (and our automated testing framework won't leave any extra resources, because it can't do this step).

Could you please try that? Then, it looks good from my end.

**Comment by entrpn** [comment]
@kweinmeister done. Thank you!

File: notebooks/community/vertex_endpoints/tf_hub_obj_detection/deploy_tfhub_object_detection_on_vertex_endpoints.ipynb [hunk 1]
```diff
@@ -1276,7 +1276,7 @@
       },
       "outputs": [],
       "source": [
-        "%%bash --out ENDPOINT_MODEL_ID\n",
+        "%%bash -s \"$ENDPOINT_ID\" --out ENDPOINT_MODEL_ID\n",
         "ENDPOINT_MODEL_ID=$(gcloud ai endpoints describe 4308018499021701120 --region=us-central1 --project=jfacevedo-project | grep \"id:\")\n",
```
Comment (kweinmeister) [inline notebooks/community/vertex_endpoints/tf_hub_obj_detection/deploy_tfhub_object_detection_on_vertex_endpoints.ipynb:1280]: Do you also want to generalize the hardcoded endpoint ID and project in this command? (And region perhaps?)

Comment (entrpn) [reply notebooks/community/vertex_endpoints/tf_hub_obj_detection/deploy_tfhub_object_detection_on_vertex_endpoints.ipynb]: done. Thanks!

File: notebooks/community/vertex_endpoints/tf_hub_obj_detection/deploy_tfhub_object_detection_on_vertex_endpoints.ipynb [hunk 2]
```diff
@@ -0,0 +1,1329 @@
+{
```
Comment (kweinmeister) [inline notebooks/community/vertex_endpoints/tf_hub_obj_detection/deploy_tfhub_object_detection_on_vertex_endpoints.ipynb:1]: <p>Apologies for mentioning 2.8 before, as we don't currently support it with a pre-built container. The notebook and cloud training versions should match.</p><p> </p><p>What about updating the tensorflow requirement above to &gt;= 2.7 and using the 2.7 container here?</p><p> </p><p>https://cloud.google.com/vertex-ai/docs/predictions/pre-built-containers</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/319/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='62'/>

Comment (kweinmeister) [inline notebooks/community/vertex_endpoints/tf_hub_obj_detection/deploy_tfhub_object_detection_on_vertex_endpoints.ipynb:1]: <p>I don't think it's essential, but we should be able to retrieve the model and endpoint, without having the user enter them manually?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/319/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='68'/>

**Comment by kweinmeister** [comment]
@entrpn Apologies, I just noticed a couple more things. I think that should do it for the review!

**Comment by entrpn** [comment]
@kweinmeister made changes requested. Thanks.

**Comment by kweinmeister** [comment]
Thanks for your contribution @entrpn !

**Review by kweinmeister — APPROVED** [review]

**Pull request merged**